doc/xcodebuild.txt

```diff
@@ -1,31 +1,139 @@
-*xcodebuild.txt* Plugin to build and test Xcode projects from vim
+*xcodebuild.txt*
+
+                            xcodebuild.vim
+                Build and test Xcode projects from Vim
 
 ==============================================================================
-CONTENTS                               *xcodebuild-contents*
+CONTENTS                                                         *XCB-Contents*
 
+      1. About.......................... |XCB-About|
+      2. Usage ......................... |XCB-Usage|
+        2.1  ............................. |XBuild|
+        2.2  ............................. |XTest|
+        2.3  ............................. |xcpretty|
+      3. Configuration ................. |XCB-Configuration|
+        3.1 .............................. |xcodebuild_run_command|
+        3.2 .............................. |xcodebuild_xcpretty_flags|
+        3.3 .............................. |xcodebuild_xcpretty_testing_flags|
 
 ==============================================================================
-INTRODUCTION                           *xcodebuild*
+ABOUT (1)                                                           *XCB-About*
```

Comment:
Mine certainly was stolen from someone else's. I think the numbers are just numbers, whereas the `~` a bit lower down does have some other effect either in just syntax highlighting, or semantic (tag?) value.